dts/bindings/sensor/st,lsm6dso-common.yaml

```diff
@@ -94,6 +94,23 @@ properties:
 
     enum: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
 
+  accel-lp-filter:
+    type: int
+    default: 0
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules